action.yml

```diff
@@ -139,36 +165,38 @@ runs:
         mask-aws-account-id: "no"
 
     - name: Store Plan
-      if: ${{ fromJSON(steps.settings.outputs.actions_enabled) }}
-      uses: cloudposse/github-action-terraform-plan-storage@v1
+      if: ${{ fromJSON(steps.settings.outputs.actions_enabled) && fromJSON(steps.atmos-plan.outputs.has_changes) }}
+      uses: cloudposse/github-action-terraform-plan-storage@added-commitsha-overwrite-input
```

Comment:
"@added-commitsha-overwrite-input" isnunclear why that version change is needed. If you can move those bits to a different pr, i'm happy to get the other bits moving